modules/ovis/include/opencv2/ovis.hpp

```diff
@@ -164,6 +164,25 @@ class CV_EXPORTS_W WindowScene {
     CV_WRAP virtual void setEntityPose(const String& name, InputArray tvec = noArray(),
                                        InputArray rot = noArray(), bool invert = false) = 0;
 
+    /**
+     * update entity animation state
+     * @param name enitity name
+     * @param animname animation state name
+     * @param offset amount of time, in seconds, to extend the animation
+     */
+    CV_WRAP virtual void updateEntityAnimation(const String& name, const String& animname,
+                                               double offset) = 0;
+
+    /**
+     * set entity animation state
+     * @param name enitity name
+     * @param animname animation state name
+     * @param enable activate or deactivate animation
+     * @param loop activate or deactivate animation loop
+     */
+    CV_WRAP virtual void setEntityAnimation(const String& name, const String& animname,
+                                            bool enable = true, bool loop = true) = 0;
```

Comment:
as a rule of thumb, if your function takes multiple boolean arguments, your API is flawed.

e.g. guess what:
```cpp
Ogre::SceneNode::_update(true, false)
```
will do?

Reply:
Thanks! I've noticed this flaw while programming and could resolve it now thanks to your suggestions! (See comments below)